spec/controllers/api/v1/movies_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe Api::V1::MoviesController, type: :controller do 
+  let!(:movie) { Movie.create(
+    title: "Star Wars",
+    summary: "Movie from childhood",
+    year: 1977
+  ) }
+
+  describe "GET#index" do
+    it "should return a list of all the movies" do
+
+      get :index
+      returned_json = JSON.parse(response.body)
+
+      expect(response.status).to eq 200
+      expect(response.content_type).to eq("application/json")
+
+      expect(returned_json.length).to eq 1
+
+      expect(returned_json[0]["title"]).to eq "Star Wars"
+      expect(returned_json[0]["summary"]).to eq "Movie from childhood"
+      expect(returned_json[0]["year"]).to eq 1977
+
+    end
+  end
+end
```

Comment:
Nice job setting up the test!  I would like to add a second `let` statement to create another movie and make sure that they are both returning.  Basically we will need some additional expect statements to ensure the second movie is returned as well